src/components/main/fundraising/Card.jsx

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable react/prop-types */
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import { ProgressBar } from './ProgressBar';
+import { Button } from '../../Button';
+import share_logo from './share_logo.svg';
+
+const StyledDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 32px;
+  gap: 16px;
+  border: 1px solid #374151;
+  border-radius: 8px;
+  background-color: #1f2a37;
+
+  @media (max-width: 1024px) {
+    padding: 10px;
+  }
+
+  @media (max-width: 768px) {
+  }
+  @media (max-width: 640px) {
+    padding: 20px;
+  }
+`;
+
+const StyledImg = styled.img`
+  width: 100%;
+`;
+const StyledH3 = styled.h3``;
+const StyledP = styled.p`
+  color: #9ca3af;
+`;
+const Buttons = styled.div`
+  display: flex;
+  gap: 16px;
+
+  @media (max-width: 640px) {
+    flex-direction: column;
+  }
+`;
+
+export const Card = ({
+  src,
+  actualSum,
+  totalSum,
+  numsOfDonors,
+  disabledButton,
+}) => {
+  const [currentAmount, setCurrentAmount] = useState(actualSum);
```

Comment:
cała ta logika chyba jeszcze nie działa prawda?

Próbowałem to odpalić ale nigdzie na stronie nie widziałem takiej możliwości. Daj znać jaki tutaj był zamysł i czy jest to dokończone